tapir/shifts/templates/shifts/shiftexemption_list.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+
+{% block content %}
+    <div class="card m-2">
+        <h5 class="card-header d-flex justify-content-between justify-content-between">
+            <span>{% trans "Shift exemptions" %}</span>
+            {% if shift_user_data %}
+                <a class="btn tapir-btn btn-outline-primary" href="{% url "shifts:create_shift_exemption" shift_user_data.id %}" id="create_shift_exemption_button">
+                    <span class="material-icons">add</span>
+                    Create
+                </a>
+            {% endif %}
+        </h5>
+        <div class="card-body">
+            <table class="table table-striped" id="shift_exemptions_table">
+                <thead>
+                <th>{% trans "User" %}</th>
+                <th>{% trans "Start date" %}</th>
+                <th>{% trans "End date" %}</th>
+                <th>{% trans "Description" %}</th>
+                <th>{% trans "Is valid" %}</th>
```

Comment:
Let's call this "Is active" instead, because it's just a temporal aspect, right?